### DISCUSSION TITLE
Exploit Prediction shows suppressed vulnerabilities

### DISCUSSION THREAD

**Discussion by JoergBruenner** [DependencyTrack/dependency-track#2536] [Q&A]
Hi,

I'm using DT version 4.71. I'm confused about the behavior of the graph in tab "Exploit 
Predictions":
The switch "show suppressed findings" seems to influence just the list below the graph.
The graph itself doesn't change if I switch supressed warnings on or off. Is this intended
behavior? I'd prefer consistence betweend graph and list.

Thanks for your help
Jörg

**Comment by nscuro** [comment]
That doesn't sound like intended behavior, do you mind [raising a defect report](https://github.com/DependencyTrack/frontend/issues/new?assignees=&labels=defect%2Cin+triage&template=defect-report.yml) for it in the frontend repository?

**Reply by JoergBruenner** [reply]
Hm, I can't reproduce it. I'm still trying ...
